test_scripts/API/VehicleData/GetVehicleData/005_GetVD_versioning.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TO ADD !!!
```

Comment:
@YarikMamykin Please describe `User story`, `Use case` and `Item` paragraphs or remove them from script description

Reply:
@aderiabin , fixed in 0c3f6df4ce34a698429a81f8a386c026c5389490